dashboard/app/views/pd/regional_partner_contact_mailer/_form_body.haml

```diff
@@ -0,0 +1,47 @@
+%p
+  %b Name:
+  - if @form[:title]
+    = "#{@form[:title]}" + ' '
+  = "#{@form[:first_name]}" + ' ' + "#{@form[:last_name]}"
+  %br
+  %b Email:
+  = "#{@form[:email]}"
+  %br
+  %b Job Role:
+  = "#{@form[:role]}"
+  %br
+  %b Job Title:
+  = "#{@form[:job_title]}"
+
+%p
+  %b School State:
+  = "#{@form[:school_state]}"
+  %br
+  %b School District:
+  - if @form[:school_district]
+    = "#{@form[:school_district]}"
+  - else
+    = "None"
+  %br
+  %b School Name:
+  - if @form[:school_name]
+    = "#{@form[:school_name]}"
+  - else
+    = "None"
+  %br
+  %b School Type:
+  = "#{@form[:school_type].capitalize}"
+  %br
+  %b Grade Levels:
+  = "#{@form[:grade_levels].join(', ')}"
+  %br
+  - if @form[:program]
+    %b Program:
+    = "#{@form[:program].map {|item| item.gsub(/\((.*)\)/, '').strip}.join(', ')}"
+
+%p
+  %b Message:
```

Comment:
I think we could leave out the message section if it's empty, but not a big deal either way

Reply:
Message was one of the fields Tanya mentioned when she commented about adding "None" if it was empty, so I'll leave it the way it is